Neos.ContentGraph.DoctrineDbalAdapter/src/ContentGraphFactory.php

```diff
@@ -46,22 +47,28 @@ public function buildForWorkspace(WorkspaceName $workspaceName): ContentGraph
             'workspace'
         ));
 
-        $row = $this->dbal->executeQuery(
-            '
-                SELECT * FROM ' . $tableName . '
-                WHERE workspaceName = :workspaceName
-                LIMIT 1
-            ',
-            [
+        $currentContentStreamIdStatement = <<<SQL
+            SELECT
+                *
+            FROM
+                {$tableName}
+            WHERE
+                workspaceName = :workspaceName
+            LIMIT 1
+        SQL;
```

Comment:
This is the first of many many places where I replaced inline SQL queries with Heredoc.
It allows the IDE to parse and highlight the queries and I even found some minor bugs/oddities this way